docs/QUnit/module.md

```diff
@@ -54,6 +54,28 @@ The module's callback is invoked with the test environment as its `this` context
 
 `QUnit.module()`'s hooks can automatically handle the asynchronous resolution of a Promise on your behalf if you return a `then`able Promise as the result of your callback function.
 
+## Exclusive tests
+
+When you are debugging your code, you will often need to _only_ run a subset of tests. You can append `only` to `QUnit.module` to specify which module you want to run.
```

Comment:
We should likely note that this is similar to other filtering methods QUnit provides.